recipe/meta.yaml

```
@@ -51,9 +51,26 @@ test:
     - pylint.reporters.ureports
     - pylint.utils
   requires:
+    - astroid >=3.2.4,<3.3.0
+    - git
+    - gitpython
     - pip
+    - py 1.11
+    - pytest 7.4
+    - pytest-benchmark 4
+    - pytest-timeout 2.2
+    - requests
+    - typing-extensions 4.11
```

```suggestion
    - py >=1.11.0,<1.12.0
    - pytest >=7.4.0,<8
    - pytest-benchmark >=4.0.0,<5.0.0
    - pytest-timeout >=2,<3
    - requests
    - typing-extensions >=4.11.0,<5
```
If the versions are strict, it will be hard to maintain this recipe, especially for new python version build-outs (e.i, py313).
I do not expect any breakages for minor versions of Python packages, except, probably, the `py` package